.github/workflows/lint.yml

```diff
@@ -26,8 +26,6 @@ jobs:
             display-os: Windows
           - feature-args: ''
```

Comment:
These were originally added in https://github.com/getsentry/sentry-cli/pull/2582, but I left these in as the empty args could be useful for having around if future flags are added so `${{ matrix.feature-suffix }}` won't need to be added below again. 

Will leave to you @szokeasaurusrex to remove if you'd prefer.